torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2481,7 +2481,9 @@ Example::
           "split_color", &::c10d::ProcessGroupNCCL::Options::split_color)
       .def_readwrite(
           "global_ranks_in_group",
-          &::c10d::ProcessGroupNCCL::Options::global_ranks_in_group);
+          &::c10d::ProcessGroupNCCL::Options::global_ranks_in_group)
+      .def_readwrite(
+          "group_name", &::c10d::ProcessGroupNCCL::Options::group_name);
```

Comment:
It seems to me this attribute would be better at the ProcessGroup level than ProcessGroupNCCL level.